pyproject.toml

```diff
@@ -69,6 +69,10 @@ dependencies = [
   "tenacity",  # retry decorator
   "sseclient-py",  # server side events for OpenAI streaming
 
+  # Web Retriever
+  "trafilatura",
+  "courlan",
```

Comment:
Let's check whether we can drop courlan

Reply:
Yes I was thinking the same; thank you for spotting this!!!

Reply:
@julian-risch update: we dropped trafilatura as it has GPL v3 licence